### PR TITLE
Storyshot Fixes

### PR DESCRIPTION
**Pull request by lalli-flores**
* IDs in Momentum UI buttons keep changing whenever a story was moved, as they are sequentially assigned, mocking the IDs so they are fixed
* Fixing Storybook so it finds Avatar images when it's hosted
* Fixed (🤞🏼) old Storybook builds not being removed from Git

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -66,7 +66,11 @@ jobs:
       - checkout_from_workspace
       - run:
           name: Build Storybook
-          command: rm -rf docs/storybook && npm run storybook:build
+          command: |
+            rm -rf docs/storybook
+            npm run storybook:build
+            git add docs/storybook
+            git commit -m "docs: build Storybook"
```
Comment (adamweeks) [inline .circleci/config.yml:73]: Committing this to the master branch will trigger another build. We'll need to add `[skip ci]`.

```suggestion
            git commit -m "docs: build Storybook [skip ci]"
```

File: src/storyshots.test.js [hunk 2]
```diff
@@ -9,6 +9,12 @@ jest.mock('./components/hooks/useActivityScroll');
 // useElementDimensions uses Resize Observer API which expects a real Element
 jest.mock('./components/hooks/useElementDimensions');
 
+// Mock react-uid to maintain a fixed ID for Momentum UI buttons
+jest.mock('react-uid', () => ({
```
Comment (adamweeks) [inline src/storyshots.test.js:13]: Awesome find!

**Review by adamweeks — CHANGES_REQUESTED** [review]
Need to update the circle config with skip ci.

File: src/storyshots.test.js (hunk 2 above)
Comment (lalli-flores) [reply src/storyshots.test.js:13]: Took me all afternoon! But worth it

**Comment by lalli-flores** [comment]
I was laying it bed last night wondering if I added the skip CI or not, I guess I didn't 😂

**Review by adamweeks — APPROVED** [review]
LGTM!

**Pull request merged**